src/cargo/ops/cargo_install.rs

```diff
@@ -270,22 +299,6 @@ fn install_one(
         ws.current()?.clone()
```

Comment:
This line seems to overwrite `pkg` with another value (the value for the workspace) for non-git sources.

This means that the second call to `make_ws_rustc_target` down below uses the new ws `pkg` instead of the original `pkg`. I *think* this is ok, based on my understanding - but wanted to follow up here w/ you @alexcrichton